airflow/models/param.py

```diff
@@ -14,7 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from typing import Any, Dict, Optional
+from typing import Any, Dict, ItemsView, MutableMapping, Optional, ValuesView
```

Comment:
Should we be getting `MutableMapping` from `collections.abc` instead? Feels a little weird to inherit from something out of typing.

Reply:
Inheriting from typing is required for generic support, I forgot until what version. Inheriting from `collections.abc` will be viable at some point, but we’re probably not there yet.

Reply:
3.9 I think (or maybe 3.8)